openmc/surface.py

```diff
@@ -139,65 +152,84 @@ def create_xml_subelement(self):
 
         element.set("type", self._type)
         element.set("boundary", self._boundary_type)
-
-        coeffs = ''
-
-        for coeff in self._coeff_keys:
-            coeffs += '{0} '.format(self._coeffs[coeff])
-
-        element.set("coeffs", coeffs.rstrip(' '))
+        element.set("coeffs", ' '.join(map(str, self._coeffs)))
```

Comment:
Actually, I don't think what you proposed will work since there is no guarantee on the ordering of `self._coeffs.values()`, but I'll manage something.